src/utils/Utils.ts

```diff
@@ -65,6 +66,19 @@ export default class Utils {
     });
   }
 
+  public static getEponymousAttributeValue(resourceInstance: ResourceInstance): string {
```

Comment:
I think it is somewhat not intuitive to have to call a `utils` function in order to get a kind of attribute.
It would be much nicer to get the eponymous attribute directly from the instance, a la `instance.getName()`.
I know, currently we only have an interface for instances. But maybe there is a way?